rest_framework/compat.py

```diff
@@ -139,16 +139,6 @@ def distinct(queryset, base):
     requests = None
 
 
-# Django-guardian is optional. Import only if guardian is in INSTALLED_APPS
-# Fixes (#1712). We keep the try/except for the test suite.
-guardian = None
-try:
-    if 'guardian' in settings.INSTALLED_APPS:
```

Comment:
Is the `INSTALLED_APPS` bit necessary? i.e. Could we not just try the `import guardian`? (And so keep this in `compat`, which is my only reservation here. Otherwise I'd be 👍)

Reply:
It would regress #1712 then.
And catching the ImproperlyConfigured error then (in a generic way) also could have the side effect of swallowing Django's own.